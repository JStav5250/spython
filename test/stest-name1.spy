def foo(int x)->int:
	return x

int y = foo(3)

foo = 5
type(foo + 3)
