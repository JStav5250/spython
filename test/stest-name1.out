TypeError: expression of type int cannot be assigned to variable 'foo' with explicit type func!!
